Makefile

```diff
@@ -103,7 +104,7 @@ clean:
 	- rm -f -r debian
 
 _unit_test: $(REPORTS)
-	gotestsum $(GO_UNITTEST_FLAGS) $(TEST) $(GINKGO_UNITTEST_FLAGS) -timeout $(TIMEOUT) || ($(MAKE) _post_unit_test && /bin/false)
+	gotestsum -- $(GO_UNITTEST_FLAGS) -timeout $(TIMEOUT) ./... || ($(MAKE) _post_unit_test && /bin/false)
```

Comment:
Don't we run the ginkgo tests anymore?

Reply:
this invocation runs both

Reply:
![image](https://github.com/flightctl/flightctl/assets/1249749/c4d636b6-5b0a-423f-a539-33e09f7eddb3)